### ISSUE TITLE
When telling Helga to mimic herself, two responses are given

### ISSUE THREAD

**Issue by bigjust** [bigjust/helga-mimic#3]
```
<qa_tester> ,mimic helga 
<helga> oh shit, we can mimic your family
<helga> i got nothing :/
```

The names of those involved have been changed to protect them.

**Comment by bigjust** [comment]
this goes for any subcommand, we need to not give a mimic response to a command that happens to mention the bot's nick

**Comment by bigjust** [comment]
fixed with e5f47273e30f1e7f751bdbb5240d164fb1324338
